README.md

```diff
@@ -83,22 +157,26 @@ Testing Private Keys
 
 **Notice: Do not use these keys in production or put funds in them. They are for testing purposes only.**
 
+### How to Use
+
+1. Deploy contracts using the master account.
+2. Interact with contracts (e.g., mint NFTs, call functions) using one of the funded test accounts.
+3. Switch wallets in your web3 wallet (e.g., MetaMask) as needed.
+
+> **Tip:** Keep deployer and user accounts separate to better understand contract permissions and simulate real dApp usage.
```

Comment:
I would also add `you risk reusing the nonce if you share the deployer private key with the browser wallet`